public_html/dashboard.js

```diff
@@ -9,6 +9,15 @@ $(document).ready(function () {
   $('#home').on('click', function () {
     $('.tab').hide();
     $('.table').empty();
+    
+    $.getJSON("/dashboard/userInfo", function (userInfo) {
+		$('#userName').html("username: " + userInfo.username);
```

Comment:
Really nitpicky but should we Uppercase the strings on the left, like Username: ?